scripts/git-hooks/pre-commit

```diff
@@ -7,9 +7,17 @@ if ! type goimports &> /dev/null; then
     GO111MODULE=off go get golang.org/x/tools/cmd/goimports
 fi
 
-# Formats each file that was changed.
+# Formats each go file that was changed.
 for file in $(git diff --cached --name-only --diff-filter=ACM | grep '\.go$')
 do
     goimports -w "${file}"
     git add "$file"
 done
+
+
+# Black linting of every python file that was changed
+for file in $(git diff --cached --name-only --diff-filter=ACM | grep '\.py$')
+do
+    black -q "$file"
+    git add "$file"
```

Comment:
I think this git add is redundant, the files are already staged

Reply:
I just checked (commit 27d8845) and without the `git add` the change made by `black` is not staged for commit

Reply:
yes, you are right, when the files are changed, they need to be staged again